lib/preparermd/overrides/environment.rb

```diff
@@ -18,7 +18,9 @@ class Index < Sprockets::Index
   def initialize(*args)
     super
 
-    @conn = Faraday.new(url: PreparerMD.config.content_store_url) do |conn|
+    opts = {url: PreparerMD.config.content_store_url}
+    opts[:ssl] = {verify: false} if !PreparerMD.config.content_store_tls_verify
```

Comment:
Right from the ["solutions to avoid"](https://github.com/lostisland/faraday/wiki/Setting-up-SSL-certificates#solutions-to-avoid) section of the Faraday docs :wink: